main/EDTypesMod.F90

```diff
@@ -265,7 +265,24 @@ module EDTypesMod
                                             ! which is used for fixation
 
      
-     
+     ! Two-stream scratch arrays
+     real(r8), allocatable :: omega_2str(:,:)   ! This is the matrix that is inverted to solve
+                                                ! the linear system of equations in the two-stream
```

Comment:
I think eventually we want to get rid of `EDTypesMod`? Is there a better place for these matrices?

Reply:
putting them on the site structure is the best spot IMO.  At the patch level, we are using up a lot of memory. At the thread (proc) level, we end up creating large scratch spaces that aren't used (but maybe thats not a big deal) for many patches. So this can go whereever the site data structure goes, or wherever we dynamically allocate stuff at the thread level (but I don't think we do the later).